src/components/Search/SearchDropdown.tsx

```diff
@@ -121,14 +130,14 @@ function SearchDropdown({
           variant='standard'
           onChange={(e) => void handleRepoSelect(e)}
         >
-          {Object.keys(repoMap).map((key) => (
+          {filteredRepoList.map((key) => (
```

Comment:
They aren't the keys anymore but the values. Maybe you can use `repo` as variable name?

Reply:
Yes, I forgot to update when changeing the mapped list. Again, not necessary since there are no duplicates.